plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/TemplateParametersPage.java

```diff
@@ -124,7 +124,13 @@ protected void doCreateControls(Composite container, DataBindingContext dbc) {
 		resetButton.setText("Reset");
 		UIUtils.setDefaultButtonWidth(resetButton);
 		resetButton.addSelectionListener(onReset());
-
+		ValueBindingBuilder
+		.bind(WidgetProperties.enabled().observe(resetButton))
+		.notUpdatingParticipant()
+		.to(selectedParameter)
+		.converting(new IsNotNull2BooleanConverter())
+		.in(dbc);
+		
```

Comment:
Everyone chills now :-) I simply copied and pasted the code from the Edit button. I didn't have to start understanding how it works.